README.md

```diff
@@ -66,6 +66,10 @@ This is the main Traversability Estimation node. It uses the elevation map and t
 
     	It is possible to subscribe to an image. The image is converted into a grayscale image and the values are mapped into an elevation map.
 
+* **`~/grid_map`** ([grid_map_msgs/GridMap])
```

Comment:
Maybe rename this to "~/initial_elevation_map" to indicate that we use this topic to initialize and that the grid_map should have an elevation layer.